### PR TITLE
Add Sudoku Solver using JavaScript

### LINKED ISSUE #984: Sudoku solver in javascript
implementation of sudoku solver in javascript
please add the Gssoc label and assign this issue to me

### PR DESCRIPTION
**Pull request by ranjal007**
Fixes #984 

#### Checklist:
- [x] 4 space indentation.
- [x] Coding conventions are followed.
- [x] Input is taken dynamically.
- [x] File names are correct.

Make sure these boxes are checked before your pull request (PR) is ready to be reviewed and merged. Thanks!

#### Changes proposed in this pull request:
-  I have created a sudoku solver

#### Languages Used:
- JavaScript
- Css
- Html

#### Files Added:
- sudoku.html
- sudoku.css
- sudoku.js
### Screenshots
WhatsApp Image 2020-03-21 at 4 33 39 PM

Thanks!

### CODE REVIEW

**Comment by asha15** [comment]
the java script code is enough. can we remove the other two files?

**Comment by ranjal007** [comment]
Yes @asha15 , U can remove the other two files. Those files are just to make the frontend of the webpage. I haven't received any points for Gssoc20 yet Hope so this PR would be merged soon.

**Comment by asha15** [comment]
files need to remove from your side and let me know

File: Sudoku_Solver/sudoku.js [hunk 1]
```diff
@@ -0,0 +1,927 @@
...
+
+    return result;
+}
+
+
+function makeItPuzzle(grid, difficulty) {
+    /*
+        difficulty:
+        
+
```
Comment (MastersAbh) [inline Sudoku_Solver/sudoku.js:187]: can you remove these blank lines

File: Sudoku_Solver/sudoku.js [hunk 2]
```diff
@@ -0,0 +1,927 @@
+﻿var table;
```
Comment (MastersAbh) [inline Sudoku_Solver/sudoku.js:1]: problem desciption at the start of file

File: Sudoku_Solver/sudoku.js [hunk 3]
```diff
@@ -0,0 +1,927 @@
...
+            var sec = timer % 60;
+            timerDiv.innerText = (("" + min).length < 2 ? ("0" + min) : min) + ":" + (("" + sec).length < 2 ? ("0" + sec) : sec);
+        }
+    }, 1000);
+}
+
+
+
+
+
```
Comment (MastersAbh) [inline Sudoku_Solver/sudoku.js:335]: ectra lines

File: Sudoku_Solver/sudoku.js [hunk 4]
```diff
@@ -0,0 +1,927 @@
...
+                break;
+            }
+            if (!unique)
+                break;
+        }
+    }
+
+ 
+    document.getElementById("game-difficulty").innerText = unique ? "Yes" : "No";
+}
```
Comment (MastersAbh) [inline Sudoku_Solver/sudoku.js:927]: Please add a blank line at the end of file.

**Comment by MastersAbh** [comment]
Hi @ranjal007 are you still working on this ?

**Comment by ranjal007** [comment]
@MastersAbh  Yes I am working on it. I am facing some difficulties in this as i had made this javascript  code  which include html and css both so after removing these files i am bit confused how to correct it.

**Comment by MastersAbh** [comment]
Just take the matrix input dynamically, rest functions should not change much I think

**Comment by MastersAbh** [comment]
Hi @ranjal007 are you working on this? If not, can we close the PR ?

**Comment by somya-kapoor** [comment]
Due to no activity for more than 7 days, I am closing this PR

**Pull request closed without merging**